cg/store/database.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, scoped_session, sessionmaker
+
+from cg.exc import CgError
+from cg.store.models import Model
+
+SESSION: Optional[Session] = None
+ENGINE = None
+
+
+def initialise_database(db_uri: str):
+    global SESSION, ENGINE
+    ENGINE = create_engine(db_uri, pool_pre_ping=True)
+    session_factory = sessionmaker(ENGINE)
+    SESSION = scoped_session(session_factory)
+
+
+def get_session() -> Session:
+    if not SESSION:
+        raise CgError("Database not initialised")
+    return SESSION()
+
+
+def get_engine():
```

Comment:
```suggestion
def get_engine():
```
type hint

Reply:
It is a ` MockConnection | Any` but I cannot import it. I think it is ok to omit (not that I have an option) since this function is only used inside this module.

Reply:
maybe we can clarify that in a docstring then so it is documented for the reader.